src/import/kv_server.rs

```diff
@@ -46,7 +46,7 @@ impl ImportKVServer {
         );
 
         let channel_args = ChannelBuilder::new(Arc::clone(&env))
-            .stream_initial_window_size(cfg.grpc_stream_initial_window_size.0 as usize)
+            .stream_initial_window_size(cfg.grpc_stream_initial_window_size.0 as i32)
```

Comment:
What if overflow?

Reply:
`grpc_stream_initial_window_size` is limited to `i32::MAX` in `Config::validate`.

Reply:
I don't see it. Can you provide a link to the sources?

Reply:
https://github.com/pingcap/tikv/pull/3142/files#diff-12d0516185b3cea2cc15d1cd46a8193aR186